vale/styles/jargonLint/Technology.yml

```diff
@@ -6,256 +6,361 @@ ignorecase: true
 level: warning
 message: Use '%s' instead of '%s'
 swap:
+  # Disable yamlint rule about line length
+  # yamllint disable-line rule:line-length
+  #
+  # Sources
+  # - https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc
+  # - https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/data.json
+  # - https://github.com/huacnlee/autocorrect/blob/main/.autocorrectrc.template
+  # - many others
+  #
+  # Restore yamlint rule about length
+  # yamllint enable rule:line-length
+  #
+  # Ad hoc: Ad hoc # risk of false positives
   ACL: ACL
   ACLs: ACLs
+  ActionScript: ActionScript
   ActiveMQ: ActiveMQ
   Adwords: Adwords
   AirPods: AirPods
   AJAX: AJAX
   Aliyun: Aliyun
   Alpine: Alpine
   Altassian: Altassian
-  Angular: Angular
-  Ansible: Ansible
-  Apache: Apache
+  Amazon S3: Amazon S3
+  Android: Android
+  Angular ?JS: AngularJS
+  Ansible(?!\.com): Ansible
+  Apache(?!\.org): Apache
   API: API
   Apify: Apify
   APIs: APIs
+  App[ ]?Store: App Store
   AppKit: AppKit
   AppleScript: AppleScript
-  App[ ]?Store: App Store
+  ASCII: ASCII
+  (?<!\.)AsciiDoc: AsciiDoc  # also a file extension
+  AST: AST
   AWS: AWS
   Bitbucket: Bitbucket
   BitTorrent: BitTorrent
-  Browserify: Browserify
-  BrowserStack: BrowserStack
+  Browserify(?!\.org): Browserify
+  BrowserStack(?!\.com): BrowserStack
   CacheStorage: CacheStorage
+  CalDAV: CalDAV
+  CC[ -]?BY[ -]?4(?:\.0)?: CC BY 4.0 # License
   CDN: CDN
-  CentOS: CentOS
-  CircleCI: CircleCI
+  CentOS(?!\.org): CentOS
+  ChatGPT: ChatGPT
+  CircleCI(?!\.com): CircleCI
   CLI: CLI
-  Cloudflare: Cloudflare
+  Cloudflare(?!\.com): Cloudflare
   CloudFront: CloudFront
-  CoffeeScript: CoffeeScript
+  CMake: CMake
+  CoffeeScript(?!\.org): CoffeeScript
   Common[ .]?js: CommonJS
-  Consul: Consul
+  CommonMark(?!\.org): CommonMark
+  Consul(?!\.io): Consul
   CORS: CORS
   CPU: CPU
   CPUs: CPUs
   (?<!\.)CSON: CSON  # also a file extension
   (?<!\.)CSV: CSV  # also a file extension
-  Cypress: Cypress
-  D3.js: D3.js
+  cURL: cURL
+  Cypress(?!\.io): Cypress
+  D3\.js: D3.js
   Dailymotion(?!\.com): Dailymotion
-  day.js: day.js
+  day\.js: day.js
   DB: DB
   DBs: DBs
-  Deno: Deno
+  DDoS: DDoS
+  Debian(?!\.org): Debian
+  Deno(?!\.com): Deno
+  DHCP: DHCP
   DLQ: DLQ
   DNS: DNS
-  Docker: Docker
+  Docker(?!\.com): Docker
   Dockerfile: Dockerfile
-  Docusaurus: Docusaurus
+  Docusaurus(?!\.io): Docusaurus
   DOM: DOM
   ECharts: ECharts
+  ECMAScript: ECMAScript
   EditorConfig: EditorConfig
   Elastic[ -]?search: Elasticsearch
-  Elasticsearch: Elasticsearch
   Electron: Electron
   Ember.js: Ember.js
   E[ -]?PUB: EPUB
   ESLint: ESLint
   Ext[ -]?3: Ext3
   Ext[ -]?4: Ext4
```

Comment:
Add Btrfs, SSD

Reply:
DAB+

MP3
MP4
MPEG
WMA
USB

Display Port
HDMI
Thunderbolt

Reply:
DAB+

MP3
MP4
MPEG
WMA
USB

Display Port
HDMI
Thunderbolt

Reply:
At this point I think we might consider categorizing this file in topics such as licenses, file extensions and protocols..

Reply:
It's something I plan to do with the Reference file. But for now, I prefer the simple approach of adding comments

Reply:
Wouldn't it be better to simply add a sources.md file in the folder? That seems easier and cleaner.

Reply:
Could we have it in line as well since this file being large also has its use. Something like:
```
swap
  # Licenses
  GPL: GPL
  MIT: MIT
```

Reply:
Yes, I plan something like this

Reply:
The licenses were moved to:
- #39

Reply:
> DAB+ DAB/DAB+
> 
> MP3 MP4 MPEG WMA USB USB Type-A (usb-a => USB t-Type-A) USB Type-B USB Type-C
> 
> Display Port HDMI Thunderbolt

This one would be moved to dedicated files